### ISSUE TITLE
Symbol's value as variable is void: org-brain-poly-hostmode

### ISSUE THREAD

**Issue by erikc96** [Kungsgeten/org-brain#320]
Hi,

I'm getting this error with the example config for using polymode in the docs.

**Comment by Kungsgeten** [comment]
Okay, that's strange. Could you try moving the `polymode` use-package declaration so that it is before the `org-brain` one? Also it would help me if you use `M-x toggle-debug-on-error` and report the result you get there.

**Comment by hjtoh** [comment]
i got the same issue too, however, after deleting the polymode use-package statement, the issue disappeared.

**Comment by benjaminwd** [comment]
Deleting all your .elc files should resolve this. There was a bug in the previous version that was giving me this error with polymode installed, but updating and recompiling seemed to resolve it for me.

**Comment by techapu** [comment]
I'm getting this error too. Where can I look? I'm not a emacs lisp expert. I had a functional debian emacs 26 installation and was working ok with org-brain. One week ago I tried guix package system. All is neat, but I tried to config my new emacs 27.1 and org-brain is guix packaged, polymode too, but it is not working. Gives me this error. I put the polymode use package declaration first, but doesn't solve it. I'm not a polymode user. Maybe polymode needs some other previous config?

**Comment by GTrunSec** [comment]
Fixed by (package! org-brain :recipe (:no-byte-compile t))

**Comment by lczch** [comment]
I guess this problem is caused by compiling autoload statements.
Remove .elc works for me.

**Comment by falematte** [comment]
I got the same problem. Any tips on how to solve this? Which .elc do I need to remove?

**Comment by hayashiya18** [comment]
`M-x package-reinstall org-brain` works for me.

**Comment by HanshenWang** [comment]
Yeah had to manually delete .elc to get poly-org working again after today's commits.

@falematte 
Directory path should look something similar to `~/.emacs.d/elpa/27.1/develop/org-brain-20210507.658`

**Comment by em-pet-dawnfawn** [comment]
@falematte  On system with a console that supports a unix `find` command, you could just delete them all, and it's pretty easy to do - `find . -name "*.elc" -exec rm {} \; `

{} stands in for the actual result from find, the one item to run the command on, and \; ends the -exec statement

**Comment by fleurc** [comment]
It's a byte-compilation error, likely the .elc compiled, adding `no-compile` fixes the issue.

If using straight with use-package just do:
`straight: (:build (:not compile))`
